code/game/objects/items/weapons/RCD.dm

```diff
@@ -451,59 +463,63 @@
 		if(checkResource(5, user))
 			to_chat(user, "Deconstructing Wall...")
 			playsound(loc, 'sound/machines/click.ogg', 50, 1)
-			if(do_after(user, 40 * toolspeed, target = A))
+			var/obj/effect/temp_visual/rcd_effect/reverse/E = new(get_turf(A))
+			if(do_after(user, 50 * toolspeed, target = A))
```

Comment:
seconds define on this delay aswell please

Reply:
should all be using seconds now